server/api/rounds.py

```diff
@@ -122,13 +122,36 @@ def cumulative_contest_results(contest: Contest) -> Dict[str, int]:
 BatchTallies = Dict[Tuple[str, str], Dict[str, Dict[str, int]]]
 
 
-def batch_tallies(election: Election,) -> BatchTallies:
+def batch_tallies(election: Election) -> BatchTallies:
+    # We only support one contest for batch audits
+    contest = list(election.contests)[0]
```

Comment:
should we throw here if `len()>1` just to prevent future bugs?

Reply:
hmm, not sure what you mean - we prevent that from happening in this PR

if you're suggesting a backup (cuz i know you like multiple layers of safety), then I'm open to it, but also we do this like 10 places in the code, so we'd have to do it everywhere. personally I'm not that worried about it - but maybe I'm just lazy. I also wrote a comment every time haha, not that it helps. i guess I could replace the comment with an `assert`...

Reply:
yeah to be clear the only thing I'm suggesting here is considering an `assert` or the like so that, when you change the *other* code, it doesn't magically look like it's working the first time you try a two-contest audit.